distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Day_Of_Week.enso

```diff
@@ -0,0 +1,52 @@
+from Standard.Base import all
+
+type Day_Of_Week
+    type Sunday
+
+    type Monday
+
+    type Tuesday
+
+    type Wednesday
+
+    type Thursday
+
+    type Friday
+
+    type Saturday
+
+    ## Convert the Day_Of_Week to an Integer
+
+       Arguments:
+       - `first_day`: The first day of the week.
+       - `include_zero`: If True, first day of the week is 0 otherwise is 1.
+    to_integer : Day_Of_Week -> Boolean -> Integer
+    to_integer self first_day=Sunday start_at_zero=False =
```

Comment:
Do we want to allow `first_day` different than `Monday` and `Sunday`?

Reply:
Allowing all as does Excel. Certainly seen Saturday as first day before.